core/src/ad-sizes.ts

```diff
@@ -241,6 +239,17 @@ const slotSizeMappings: SlotSizeMappings = {
 			adSizes.fluid,
 		],
 	},
+	'comments-expanded': {
+		mobile: [
```

Comment:
Do we need to define any tablet or desktop sizes for this ad slot?

Reply:
I think that the mobile definition of slots encompasses breakpoints for mobile devices and above. The same pattern is used for carrot, right and the merchandising slots